File: 485-max-consecutive-ones/max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let ans=[]
    let c=0
    for (let i=0;i<nums.length;i++){
        if (nums[i]==1){
            c+=1
        }else{
            ans.push(c)
            c=0
        }

    }
    ans.push(c)
    return Math.max(...ans)
};

File: 485-max-consecutive-ones/max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        ans = []
        c=0
        for i in nums:
            if i == 1:
                c+=1
            else:
                ans.append(c)
                c=0
        ans.append(c)
        return max(ans)